setup.cfg

```diff
@@ -0,0 +1,74 @@
+[metadata]
+name = geocat.viz
+version = attr: setuptools_scm.get_version
+author = GeoCAT Team
+author_email = geocat@ucar.edu
+description= GeoCAT-viz is vizualization component of the GeoCAT project and
+             provides implementations of convenience functions for publication-ready plotting
+             of geosciences data
+long_description_content_type=text/markdown
+long_description = file: README.md
+url = https://geocat-viz.readthedocs.io
+license = Apache-2.0
+license_files = file: LICENSE
+classifiers =
+    License :: OSI Approved :: Apache Software License
+    Operating System :: OS Independent
+    Intended Audience :: Science/Research
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Topic :: Scientific/Engineering
+
+[options]
+zip_safe = False
+include_package_data = True
+python_requires = >=3.8, <3.11
+package_dir =
+    =src
+packages =
+    geocat
+    geocat.viz
+setup_requires =
+    setuptools_scm
+    setuptools
+    pip
+    geocat.f2py
+install_requires =
+    cf_xarray>=0.3.1
+    cftime
+    eofs
+    geocat.f2py
+    metpy
+    scipy
+    xarray
+    xskillscore
+    packaging
+tests_require =
+    pytest
+    netcdf4
+    geocat-datafiles
```

Comment:
I think you've just pulled this from comp, but it should be worth going through the original requirements.txt and meta.yml to figure out which of these you actually need

Reply:
Oops done